1_portfolio_check_initialisation.R

```diff
@@ -30,16 +30,31 @@ source("0_portfolio_input_check_functions.R")
 source("0_global_functions.R")
 source("0_sda_approach.R")
 
-project_name <- "liechtenstein_p2020"
-twodii_internal <- TRUE
-# TRUE or FALSE: TRUE means that the code is running on a 2dii laptop with dropbox connection
-
-#####################################################################
-###ONLY FOR EXTERNAL PROJECTS (twodii_internal <- FALSE):
-# Variables must exist for internal projects
-project_location_ext <- "C:/Users/clare/Desktop/ExternalTest"
-data_location_ext <- "C:/Users/clare/Desktop/Git/pacta-data/2019Q4"
-#####################################################################
+
+# Set Project Settings
+
+# within the "project_setting.yml" config file, set the project_name, the twodii_internal switch,
+# and the external data locations, if necessary.
+# the project_name will determine the name of the folder that is created for this project
+# Set twodii_internal to TRUE to run the analysis on an internal 2dii laptop
+# This setting uses the dropbox connection for data import
+# Set twodii_internal to FALSE, tu use external data locations
+# Specify these data locations in the config file "project_settings.yml" in the repo
+
+cfg <- config::get(file = "project_settings.yml")
+stopifnot(exists("cfg")  == T)
+stopifnot(cfg %>% class() == "list")
+stopifnot(cfg %>% length() == 2)
```

Comment:
this could be a function, something like: 

``` r 
check_valid_cfg <- function(cfg){
  stopifnot(exists("cfg")  == T)
  stopifnot(cfg %>% class() == "list")
  stopifnot(cfg %>% length() == 2)
  
  invisible(cfg)
}
``` 

would make it easier to add more checks in the future, and keep them organized

Reply:
sure, will add this

Reply:
changes pushed, pls review
@jdhoffa @Clare2D